### DISCUSSION TITLE
下划线输入的时候能不能不要长度不变

### DISCUSSION THREAD

**Discussion by Chendamao** [wangeditor-next/wangEditor-next#371] [General]
![17321515667274](https://github.com/user-attachments/assets/bf83a7d8-3b96-4ae2-a135-d17cce64c2f0)
![17321516041411](https://github.com/user-attachments/assets/864b5e96-d888-4821-bc2e-b3ac83859c57)

就类似图片这种，输入内容的时候下划线不要延长

**Comment by cycleccc** [comment]
这个功能 wangEditor 内置不太合适，试试监听 setHtml 删除后面多余的下划线空格吧。

**Comment by cycleccc** [comment]
该展示需求不纳入内置能力，已有 setHtml 后清理额外下划线空格的替代方案。本讨论现关闭。
